```
======================================================================
ERROR: couchexport.tests.test_writers:CsvFileWriterTests.test_csv_file_writer_bom
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/couchexport/tests/test_writers.py", line 64, in test_csv_file_writer_bom
    writer.write_row(headers)
  File "corehq/ex-submodules/couchexport/writers.py", line 115, in write_row
    row if isinstance(row, six.text_type) else row.decode('utf-8')
AttributeError: 'list' object has no attribute 'decode'
----------------------------------------------------------------------
```

--- a/corehq/ex-submodules/couchexport/writers.py
+++ b/corehq/ex-submodules/couchexport/writers.py
@@ -112,7 +112,8 @@ class CsvFileWriter(ExportFileWriter):
 
     def write_row(self, row):
         self._csvwriter.writerow([
-            row if isinstance(row, six.text_type) else row.decode('utf-8')
+            col if isinstance(col, six.text_type) else col.decode('utf-8')
+            for col in row
         ])
 
 

